db-scripts/createdb.sql

```diff
@@ -57,63 +57,12 @@ create table if not exists equipment_item(
    FOREIGN KEY(equipment_type) REFERENCES equipment_rental_type(id) on update cascade
 );
 
-create table if not exists equipment_rentals(
-   item_id integer not null,
-   user_id integer not null,
-   date_borrowed date not null default current_date,
-   return_date date not null,
-   price money not null,
-   condition equipment_condition_enum not null default 'ready',
-   PRIMARY KEY(user_id, item_id),
-   FOREIGN KEY(item_id) REFERENCES equipment_item(id) on update cascade,
-   FOREIGN KEY(user_id) REFERENCES users(id) on update cascade
-);
```

Comment:
don't delete this table. also change user_id fk to have 'on delete set null' and also make user_id column to be nullable.

Reply:
oops done